gke-scripts/WorkloadCertificateConfig.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: security.cloud.google.com/v1alpha1
+kind: WorkloadCertificateConfig
+metadata:
+  name: default
+spec:
+  # Required. The SPIFFE trust domain. This must match your clusters
+  # Workload Identity pool.
+  trustDomain: ${PROJECT_ID}.svc.id.goog
+
+  # Required. The CA service that issues your certificates.
+  certificateAuthorityConfig:
+    certificateAuthorityServiceConfig:
+      endpointURI: ${SUBORDINATE_CA_URI}
```

Comment:
The definition of SUB_CA_URI in 00-common-env.sh is the old beta API of CAS. Can we update that to reflect the new GA API?

Reply:
What should be the new value?

Reply:
`//privateca.googleapis.com/projects/PROJECT_ID/locations/ISSUING_CA_POOL_LOCATION/caPools/ISSUING_CA_POOL_NAME`

Reply:
Done.